### DISCUSSION TITLE
Server Action works just with input field?!

### DISCUSSION THREAD

**Discussion by NicoCaldo** [vercel/next.js#60444] [Help]
### Summary

I'm using shadcn-ui and I've noticed that using the form with a server action like `<form action={async (form: FormData) => { .....  >`
form always have just the input field. I have `radio/slider/toggle` and no one of them is read.

I've also noticed that all the examples in the [documentation](https://nextjs.org/docs/app/building-your-application/data-fetching/server-actions-and-mutations) always have just one input field. 

### Example

https://github.com/shadcn-ui/ui/issues/2356

**Comment by ananuness** [comment]
Hii! As you can see there in [Behaviors](https://nextjs.org/docs/app/building-your-application/data-fetching/server-actions-and-mutations#behavior), Server actions works with any actions like:

- Form action prop
- Event handlers
- useEffect
- And others form elements like button

Hope it helps!

**Reply by NicoCaldo** [reply]
thanks for your answer but it does not answer why I'm seeing just the first input submitted to the server action

**Reply by ananuness** [reply]
I'm sorry, I read it too fast 😅 in this case you need to provide how your server action and form structure is written to see the problem!
